satpy/etc/readers/tropomi_l2.yaml

```diff
@@ -42,16 +42,16 @@ datasets:
         file_type: tropomi_l2
         file_key: 'PRODUCT/SUPPORT_DATA/GEOLOCATIONS/longitude_bounds'
         standard_name: assembled_longitude_bounds
-    offset_time:
-        name: 'offset_time'
+    delta_time:
+        name: 'delta_time'
         file_type: tropomi_l2
         file_key: 'PRODUCT/delta_time'
-        standard_name: offset_time
-    ref_time:
-        name: 'ref_time'
+        standard_name: delta_time
+    time:
+        name: 'time'
         file_type: tropomi_l2
         file_key: 'PRODUCT/time'
-        standard_name: ref_time
+        standard_name: time
```

Comment:
Isn't there a more CF standard name for time? What are the units of this dataset?

Reply:
Units is `  :units = "seconds since 2010-01-01 00:00:00";`. The standard name is `time`.
```
int time(time=1);
  :units = "seconds since 2010-01-01 00:00:00";
  :standard_name = "time";
  :axis = "T";
  :long_name = "reference time for the measurements";
  :comment = "The time in this variable corresponds to the time in the time_reference global attribute";
  :_FillValue = -2147483647; // int
  :_ChunkSizes = 1U; // uint
```